.github/workflows/yocto-build-deploy.yml

```diff
@@ -1065,13 +1065,16 @@ jobs:
 
     steps:
       # https://github.com/actions/create-github-app-token
+      # We give the owner arg, meaning that this token will be valid for all repositories in the org
+      # This behvaiour is required for private submodules
       - name: Create GitHub App installation token
         uses: actions/create-github-app-token@5d869da34e18e7287c1daad50e0b8ea0f506ce69 # v1.11.0
         id: app-token
         if: vars.FLOWZONE_APP_ID != ''
         with:
           app-id: ${{ vars.FLOWZONE_APP_ID }}
           private-key: ${{ secrets.GH_APP_PRIVATE_KEY }}
+          owner: ${{ github.repository_owner }}
```

Comment:
I think the owner is the current org by default, does this actually change anything?

Reply:
@klutchell if no `owner` arg is provided the docs say it defaults to repo only scope: https://github.com/actions/create-github-app-token/tree/5d869da34e18e7287c1daad50e0b8ea0f506ce69/?tab=readme-ov-file#create-a-token-for-the-current-repository

Logs on the step in action also show this:

"Run actions/create-github-app-token@5d869da34e18e7287c1daad50e0b8ea0f506ce69
owner and repositories not set, creating token for the current repository ("balena-raspberrypi")"

Reply:
TIL, LGTM

Reply:
actually wait, it looks like my `owner` arg is ignored in this PR run

Reply:
ignore me ^ I put it in the wrong place, (leviathan token) , fixing

Reply:
ok confirmed, now we get an org wide token

```
Run actions/create-github-app-token@5d869da34e18e7287c1daad50e0b8ea0f506ce69
repositories not set, creating token for all repositories for given owner "balena-os"
```

Reply:
will test on a device repo